ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -582,6 +582,7 @@ DFSS:
   - discharge_fan_lost_power_alarm
   - schedule_run_command
   - discharge_fan_run_time_accumulator
+  - discharge_fan_speed_mode
```

Comment:
move this to DFVSC, there shouldnt be any speed reference on DFSS.

Reply:
Moved to DFVSC